packages/mui-codemod/src/v5.0.0/base-remove-component-prop.test/actual.tsx

```diff
@@ -0,0 +1,25 @@
+// @ts-nocheck
+<Input component={CustomRoot} />;
```

Comment:
We should not depend on the name of the component. It would be best to depend on the import.

Reply:
Can you elaborate? If you are talking about filtering base components, I changed the logic so we filter base components based on import statements.